flow/network.h

```diff
@@ -646,4 +648,123 @@ class INetworkConnections {
 	// Returns the interface that should be used to make and accept socket connections
 };
 
+struct DelayGenerator : FastAllocated<DelayGenerator> {
```

Comment:
This class needs a comment block explaining what the model is and how it is meant to be used.

`FastAllocated` is not useful here as this class is never dynamically allocated and there is only one per process anyways.